tests/common/dualtor/dual_tor_utils.py

```diff
@@ -929,7 +930,7 @@ def _generate_hashed_ipv6_packet(src_mac, dst_mac, dst_ip, hash_key):
     if len(generate_hashed_packet_to_server.packets_cache[call_signature]) < count:
         pkt_num = count - len(generate_hashed_packet_to_server.packets_cache[call_signature])
         for _ in range(pkt_num):
-            if ipaddress.ip_address(target_server_ip.decode()).version == 4:
+            if ipaddress.ip_address(six.text_type(target_server_ip)).version == 4:
```

Comment:
Probably do a search in to code base to see if there is similar issue in other files?

Reply:
Will do

Reply:
Search code base and change another two places.
There are a few other code using ip_address() with decode() but fixed by Xiawei Jiang.